gulpfile.babel.js

```diff
@@ -290,6 +292,18 @@ function lintBootstrap() {
 		}));
 }
 
+export function lintMarkup() {
+	if (isProdBuild()) {
+		return gulp.src([...settings.sources.markup, './*.html'])
```

Comment:
There are no html files in the root directory. 

So it should be:
```
gulp.src(settings.sources.markup)
```
instead of:
```
gulp.src([...settings.sources.markup, './*.html'])
```